spring-cloud-gcp-autoconfigure/src/test/java/com/google/cloud/spring/autoconfigure/core/UserAgentHeaderProviderIntegrationTests.java

```diff
@@ -14,10 +14,11 @@
  * limitations under the License.
  */
 
-package com.google.cloud.spring.core;
+package com.google.cloud.spring.autoconfigure.core;
```

Comment:
I imagine you'd want to keep this in the `core` package, no? I'm still trying to explore why it works in autoconfigure and not in core

Reply:
Because it gets the package name from MANIFEST, and that's not available while the same package is being built.

Reply:
I just put it there because that's where tests for core go in the autoconfigure module.